lib/util/git/url.spec.ts

```diff
@@ -144,5 +174,14 @@ describe('util/git/url', () => {
         'https://gitlab-ci-token:token@gitlab.com/some/repo.git'
       );
     });
+
+    it('returns https url with encoded github token', () => {
+      hostRules.find.mockReturnValueOnce({
+        token: 'token',
+      });
+      expect(getRemoteUrlWithToken('ssh://github.com/some/repo.git')).toBe(
+        'https://x-access-token:token@github.com/some/repo.git'
```

Comment:
is this new behavior? or does it return https url before too?

if it changed, i consider this a breaking, as some people use ssh with git-refs datasource

Reply:
this is not new behaviour - you can see in the tests on line 143 and line 173 of this file that this was already the case